09-intermediate-rails/testing-auth.md

```diff
@@ -32,54 +62,141 @@ class ActiveSupport::TestCase
 end
 ```
 
-The above code will populate the OmniAuth request data hash with the information that you have provided here. This is especially useful and necessary when you are testing controller action that may require a user to be logged in.
+The above code will populate the **default** OmniAuth request data hash with the information that you have provided here. This is especially useful and necessary when you are testing controller actions that may require a user to be logged in. This is mock data which should match what we may expect to see from the _real_ `auth_hash` we'll get back from GitHub. This is setting up the **default** data and nothing more. We'll need to _use_ this data in our tests in the next step.
+
+#### Session Tests
 
-#### In your Test-cases
+First, let's create a method which will allow us to log in using the default OmniAuth data that we just configured.
+
+```ruby
+# sessions_controller_test.rb
+describe SessionsController do
+  def login
```

Comment:
This method could be moved to the `test_helper.rb`  So that it's shared between controller tests.